.storybook/webpack.config.js

```diff
@@ -84,6 +84,7 @@ module.exports = (baseConfig, env) => {
       extensions: [".mjs", ".js", ".jsx", ".ts", ".tsx"],
       alias: {
         sharify: sharifyPath.replace(/\.js$/, ""),
+        "styled-components": path.resolve("./node_modules/styled-components"),
```

Comment:
Why is this needed?

Reply:
It’s a multiple copies of lib error due to yarn link